packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -35,39 +35,29 @@ abstract class OverlayRoute<T> extends Route<T> {
     super.install(insertionPoint);
   }
 
-  /// Controls whether [didPop] calls [finished].
+  /// Controls whether [didPop] calls [NavigatorState.finalizeRoute].
   ///
   /// If true, this route removes its overlay entries during [didPop].
-  /// Subclasses can override this getter if they want to delay the [finished]
-  /// call (for example to animate the route's exit before removing it from the
+  /// Subclasses can override this getter if they want to delay finalization
+  /// (for example to animate the route's exit before removing it from the
   /// overlay).
```

Comment:
a question that this text raises (and should answer) is: if they do, are they required to call finalizeRoute?